src/content/json/header-assets.js

```diff
@@ -0,0 +1,49 @@
+import blueprintIcon from "../images/blueprint_banner_negative.webp";
+import styled from 'styled-components';
+
+export const BlueprintIcon = () => {
+  return (
+    <img style={{ maxWidth: '150px' }} src={blueprintIcon}></img>
```

Comment:
We should have an `alt` prop here if the user can't load the image for some reason.

It'd also be cool if the icon re-directed the user to the home page on click.

Reply:
This icon should also have some left margin of say, `50px` (this value is used again in a comment about the `StyledHeaderList`

Reply:
You can also just replace the last seven characters with `/>`, so the `img` JSX will look like: `<img style=... src=... alt=... />`